packages/components/psammead-sitewide-links/src/List/index.jsx

```diff
@@ -68,7 +67,7 @@ const StyledListItem = styled.li`
 const List = ({ links }) => (
   <StyledList role="list">
     {links.map(link => (
-      <StyledListItem key={nanoid()} role="listitem">
+      <StyledListItem key={link.id || JSON.stringify(link)} role="listitem">
```

Comment:
Based on the [discussion in this PR](https://github.com/bbc/simorgh/pull/1879#discussion_r292405198) I think we are going with just using the text as a key.